ingest/ingest-publications.py

```diff
@@ -320,34 +334,36 @@ def process_publication(publication, endpoint='http://localhost:2020/ds/sparql')
     return [json.dumps(get_metadata(es_id)), json.dumps(pub)]
     fidx.close()
 
-def generate(threads, sparql):
+def generate(threads):
     pool = multiprocessing.Pool(threads)
     params = [pub for pub in g1.subjects(RDF.type, BIBO.Document)]
     print("params: ", params)
     return list(chain.from_iterable(pool.map(process_publication, params)))
 
-get_orgs_query = load_file("queries/listOrgs.rq")
-get_subjects_query = load_file("queries/listSubjects.rq")
-get_author_query = load_file("queries/listAuthors.rq")
-get_pub_query = load_file("queries/listPubs.rq")
-
-g1 += describe(endpoint,get_orgs_query)
-g1 = g1 + describe(endpoint,get_subjects_query)
-g1 = g1 + describe(endpoint,get_author_query)
-g1 = g1 + describe(endpoint,get_pub_query)
-
 if __name__ == "__main__":
     parser = argparse.ArgumentParser()
-    parser.add_argument('--threads', default=12, help='number of threads to use (default = 6)')
-    parser.add_argument('--sparql', default='http://localhost:2020/ds/sparql', help='sparql endpoint')
+    parser.add_argument('--threads', default=16, help='number of threads to use (default = 12)')
+    parser.add_argument('--sparqlendpoint', default='http://yourhost:8780/vivo/api/sparqlQuery', help='sparql endpoint')
```

Comment:
Is the "yourhost:8780" an example or hardcoded value? Maybe add a comment inline here to call that out?

Reply:
this is an example because the code is public, the actual URL is in the secrets file

Reply:
added more info in the help for the --sparql parameter